config/application.rb

```diff
@@ -115,7 +115,7 @@ class Application < Rails::Application
     config.autoload_paths += %W(#{config.root}/app/models/compute_resources)
     config.autoload_paths += %W(#{config.root}/app/models/fact_names)
     config.autoload_paths += %W(#{config.root}/app/models/lookup_keys)
-    config.autoload_paths += %W(#{config.root}/app/models/host_status)
+    # config.autoload_paths += %W(#{config.root}/app/models/host_status)
```

Comment:
This is from previous commits. My guess is we can remove this line completely.

Reply:
and all the above IMHO. we should not add subfolders of `app/*` as all the folders are roots for autoloading and if the files follow the naming patterns expected by autoloaders, it should work without this.